otx/algorithms/action/adapters/mmaction/__init__.py

```diff
@@ -4,6 +4,9 @@
 # SPDX-License-Identifier: Apache-2.0
 
 from .data import OTXRawframeDataset
+from .models.backbones import MoViNetBase
+from .models.heads import MoViNetHead
+from .models.recognizers import Recognizer3D
 from .utils import patch_config, set_data_classes
 
-__all__ = ["OTXRawframeDataset", "patch_config", "set_data_classes"]
+__all__ = ["OTXRawframeDataset", "patch_config", "set_data_classes", "MoViNetBase", "MoViNetHead", "Recognizer3D"]
```

Comment:
Is it appropriate for the model to be initialized here?

Reply:
It is initialization adapters for mmaction. If movinet is adpated from mmaction, then it might be okay